.gitignore

```diff
@@ -12,3 +12,6 @@ build/
 *.gcda
 *.gcno
 *.gcov
+
+# Ignore certificates.
+*.crt
```

Comment:
Since all the creds are put into `certificates` directory, would it be better to ignore that directory as well: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/demos/CMakeLists.txt#L41

The reason is that sometimes the extension is `.pem` or `.crt.txt`.

Reply:
Does the following file not ignore certs: https://github.com/aws/aws-iot-device-sdk-embedded-C/blob/development/demos/certificates/.gitignore